lib/msal-core/test/UserAgentApplication.spec.ts

```diff
@@ -34,6 +35,9 @@ describe("UserAgentApplication.ts Class", function () {
     const TEST_REDIR_URI = "https://localhost:8081/redirect.html";
     const TEST_LOGOUT_URI = "https://localhost:8081/logout.html";
     const TEST_ERROR_CODE = "error_code";
+    const TEST_INTERACTION_REQ_ERROR_CODE = "interaction_required";
+    const TEST_LOGIN_REQ_ERROR_CODE = "login_required";
+    const TEST_CONSENT_REQ_ERROR_CODE = "consent_required";
```

Comment:
Do these need to be duplicated here, or could you pull in this? https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/f8e0eb4c11cb355212c2fa2d2c158a985d748326/lib/msal-core/src/error/InteractionRequiredAuthError.ts#L6-L16

Reply:
No that's a good point. I will do that.

Reply:
Done, please check and resolve if it looks good.

Reply:
Lgtm.